src/java/org/apache/cassandra/db/commitlog/CommitLog.java

```diff
@@ -112,6 +114,11 @@ CommitLog start()
         return this;
     }
 
+    public boolean hasFilesToReplay()
+    {
+        return new File(segmentManager.storageDirectory).listFiles(unmanagedFilesFilter).length > 0;
```

Comment:
IDEA complains that this:

new File(segmentManager.storageDirectory).listFiles(unmanagedFilesFilter)

can produce NPE.

You should rewrite this to use Cassandra's "File", not Java's one. 

I know this is branch 3.11 where that stuff is not in yet but IDE complaining about NPE should be fixed anyway somehow I guess.